pkg/sources/generator/tickgen.go

```diff
@@ -114,7 +114,9 @@ func WithReadTimeOut(timeout time.Duration) Option {
 
 // NewMemGen fuction creates an instance of generator.
 // ctx  - context passed by the cmd/start.go a new context with cancel
-//        is created for use by this vertex.
+//
```

Comment:
is this intended? i think that comment was part of prev line

Reply:
It's go 1.19 format.